package.json

```diff
@@ -67,7 +67,7 @@
     "@lukeed/uuid": "^2.0.1",
     "@popperjs/core": "^2.11.7",
     "@radix-ui/react-checkbox": "^1.0.4",
-    "@radix-ui/react-select": "^1.2.2",
+    "@radix-ui/react-select": "^2.0.0-rc.7",
```

Comment:
Are we ok with using a prerelease version? Why do you need it now?

Reply:
I asked @Xstoudi  to check if the behavior in v2 is still the same regarding a trick that was used to force radix-ui's select component to be controlled when the value is `undefined`.

But beyond this check we can go back to v1 to stay on the safe side.